src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task {
+
+    protected String from;
+    protected String to;
```

Comment:
"from" and "to" might be ambiguous, well at least from my pov. Instead of time, readers could think of it in terms of place.
Other than that, nice code 👍🏻 good to go.

Reply:
Thanks for the feedback! You're right, will adjust as necessary 👍🏼